docs/enterprise/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: lakeFS Enterprise
+description: lakeFS Enterprise is an enterprise-ready lakeFS solution, implemented using our best practices, providing additional features, such as RBAC, SSO and Support SLA.
+nav_order: 81
+has_children: true
+---
+
+# lakeFS Enterprise
+
+{: .no_toc }
+
+lakeFS Enterprise is an enterprise-ready lakeFS solution, implemented using our best practices, providing additional features, such as:
+
+## Features only available in lakeFS Enterprise
+* RBAC
+* SSO
+* Support SLA
```

Comment:
```suggestion
lakeFS Enterprise is an enterprise-ready lakeFS solution that provides a support SLA and additional features to the open-source version of lakeFS. The additional features are: 

* [RBAC](/reference/rbac.html)
* [SSO](/enterprise/sso.html)
```

I would also add a H2 section called "Support SLA" and describe what kind of SLAs we offer, what the support routes are — or _some_ kind of verbiage that would be useful :)